src/App.js

```diff
@@ -1,23 +1,52 @@
-import React, { useState } from 'react';
-import FileUpload from './components/FileUpload';
-import axios from 'axios';
+import React, { useState, useEffect } from 'react';
+import { languageOptions, contentVariations } from './LanguageContent';
+import { ReactComponent as LanguagePicker } from './components/LanguagePicker.svg';
+import HubspotForm from './components/HubspotForm';
 
 function App() {
 
-  const [phoneNumber, setPhoneNumber] = useState('');
-  const [document, setDocument] = useState('');
+  const [language, setLanguage] = useState('en')
+  
+  
+  const browserLanguage = window.navigator.userLanguage || window.navigator.language;
+   
+  useEffect(() => {
+    if (browserLanguage){
+      setLanguage(browserLanguage.substring(0,2))
+    }
+  },[browserLanguage])
 
-  const submitForm = (user) => {
-    axios.post('http://localhost:5000/api/users/add', user)
-      .then(response => {
-        console.log('response:', response.data)
-      })
-  };
+  console.log('browserLanguage :>> ', browserLanguage);
 
+  
   return (
-    <div>
-      <h1>e-immigrate</h1>
-      <FileUpload submitForm={submitForm} phoneNumber={phoneNumber} setPhoneNumber={setPhoneNumber} document={document} setDocument={setDocument} />
+    <div className='container d-flex flex-column align-items-center justify-content-center pt-4'>
```

Comment:
We will go over these since we will not use bootstrap, does wayyy too much and always causes headaches down the line, just CSS and most of all CSS Grid :)